pkg/resources/core/v1/namespace/projectannotations.go

```diff
@@ -23,8 +25,10 @@ type projectNamespaceAdmitter struct {
 	sar authorizationv1.SubjectAccessReviewInterface
 }
 
-// Admit ensures that the user has permission to change the namespace annotation for
-// project membership, effectively moving a project from one namespace to another.
+// Admit ensures that the:
+//   - user has permission to change the namespace annotation for project membership, effectively moving a project from
+//     one namespace to another.
+//   - deletion of `local` and `fleet-local` namespace is not allowed as it could
```

Comment:
as it could......what?

Reply:
Lol. 😂 
I thought I had removed those words.

Reply:
Done. Removed.